release/docker/opendistroforelasticsearch-data-prepper-0.7.1-alpha.Dockerfile

```diff
@@ -1,6 +1,4 @@
-# building on CentOS 7 to be in line with
-# https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/elasticsearch
-
+# For the v0.7.0-alpha release we will use the amazoncorretto amazon linux.
```

Comment:
update this comment to make it release neutral

Reply:
Will remove it in a follow-up PR (the `FROM` line is self-explanatory)